### PR TITLE
CB-11926 Pass the file transfer server address to the tests via variable

### PR DESCRIPTION
**Pull request by alsorokin**
https://issues.apache.org/jira/browse/CB-11926

### CODE REVIEW

File: lib/PluginsManager.js [hunk 1]
```diff
@@ -41,16 +43,31 @@ PluginsManager.prototype.installTestsForExistingPlugins = function () {
     installedPlugins.forEach(function(plugin) {
         // there is test plugin available
         if (fs.existsSync(path.join(plugin.dir, 'tests', 'plugin.xml'))) {
-            me.installSinglePlugin(path.join(plugin.dir, 'tests'));
+            var additionalArgs = '';
+            if (plugin.id.indexOf('cordova-plugin-file-transfer') >= 0) {
+                var fileServerUrl = Server.prototype.getConnectionAddress.bind({
```
Comment (vladimir-kotikov) [inline lib/PluginsManager.js]: Can we just create a new `Server()` and call `getConnectionAddress` on the instance instead? IMO this seems to be more intuitive than using `bind`

File: lib/PluginsManager.js [hunk 2]
```diff
@@ -41,16 +43,31 @@ PluginsManager.prototype.installTestsForExistingPlugins = function () {
...
     this.showPluginsVersions();
 };
 
 PluginsManager.prototype.installSinglePlugin = function (plugin) {
-    if (fs.existsSync(path.resolve(this.storedCWD, plugin))) {
-        plugin = path.resolve(this.storedCWD, plugin);
+    var pluginPath = plugin;
+    var args = '';
+    // separate plugin name from args
+    var argsIndex = plugin.indexOf(' --');
```
Comment (vladimir-kotikov) [inline lib/PluginsManager.js]: So you're building a string in `installTestsForExistingPlugins` and then splitting it back to `(plugin, arguments)` tuple? I don't think this is an efficient way to pass additional arguments to the function. Can we just add an optional argument to be able to specify any flags to `cordova plugin add`?

Comment (alsorokin) [inline lib/PluginsManager.js]: Paramedic doesn't have the functionality to specify arguments to pass with `plugin add`, so this is done to support a scenario when the user specifies a plugin name with variables.

Comment (alsorokin) [inline lib/PluginsManager.js]: So basically this logic is still needed but can be moved out of this function to `paramedic.js`
Which option is preferable?

Comment (vladimir-kotikov) [inline lib/PluginsManager.js]: Ah, I see. I think you can leave it here but it's worth documenting. Also please add a note to 'paramedic.js', clarifying that we now support passing args to `cordova plugin add`

**Pull request merged**